### PR TITLE
ROS2 sentence message, updated package.xml and cmakelist

### PR DESCRIPTION
**Pull request by klintan**
Need to change the other messages, so will start this as a draft, and try to finish it up asap :)

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -8,12 +9,20 @@
   <url type="website">http://ros.org/wiki/nmea_msgs</url>
   <author email="eric@ericperko.com">Eric Perko</author>
```
Comment (evenator) [inline package.xml]: Feel free to add yourself as an author :)

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-add_message_files(
-  FILES
-  Gpgga.msg
-  Gpgsa.msg
-  Gpgsv.msg
-  GpgsvSatellite.msg
-  Gprmc.msg
-  Sentence.msg
-)
+find_package(tinyxml2 REQUIRED)
```
Comment (evenator) [inline CMakeLists.txt]: I can't get this to build in a minimal docker container. Here's what I did:

```bash
docker pull osrf/ros:crystal-desktop
docker run -it osrf/ros:crystal-desktop
apt-update && apt-install python3-colcon-common-extensions
mkdir -p code/ros2_ws/src
cd ros2_ws/src
git clone --branch ros2 https://github.com/klintan/nmea_msgs.git
cd ..
rosdep install --from-paths src
colcon build
```

The build fails with a missing dependency: tinyxml2

When I remove this `find_package`, the build succeeds. I don't think you need this dependency for anything, do you?

**Review by evenator — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (klintan) [reply CMakeLists.txt]: Yeah it wasn't working on my local mac without it, but I also thought it was really strange. So I will remove it.

**Comment by klintan** [comment]
OK updated, let me know if it checks out. I'll make it into a proper PR, feel free to merge if everything looks ok.

**Review by evenator — APPROVED** [review]

**Comment by evenator** [comment]
This looks good to me. Thanks for contributing! I'm going to look into what it'll take to get this building on the ROS build farm so we have ROS 2 binary packages.

**Pull request merged**